cmd/osbuild-worker-executor/handler_build.go

```diff
@@ -84,11 +84,12 @@ func runOsbuild(buildDir string, control *controlJSON, output io.Writer) (string
 	// #nosec G204
 	cmd = exec.Command(
 		"tar",
+		"-C",
```

Comment:
This change will conflicts with https://github.com/osbuild/osbuild-composer/pull/4193 afaict (and that PR probably needs adjusting)  - (sorry!) it's not really important, the fix is trivial